demo/fhir/config/default/fhir-server-config.json

```diff
@@ -2,13 +2,10 @@
   "__comment": "config for internal fhir-server deployments",
   "fhirServer": {
     "core": {
-      "tenantIdHeaderName": "X-FHIR-TENANT-ID",
-      "datastoreIdHeaderName": "X-FHIR-DSID",
-      "checkReferenceTypes": true,
       "conditionalDeleteMaxNumber": 1,
       "defaultPrettyPrint": true,
-      "originalRequestUriHeaderName": "X-Watson-DP-URL-in",
-      "serverRegistryResourceProviderEnabled": false
+      "serverRegistryResourceProviderEnabled": false,
+      "ifNoneMatchReturnsNotModified": true
```

Comment:
intended?

Reply:
ah, forgot i had this change sitting around.  i think its fine to commit though...mostly its just removing cruft where we were explicitly setting config params to their default value